stock_move_backdating/wizard/__init__.py

```diff
@@ -17,4 +17,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from . import stock_partial_picking
+

```

Comment:
Well, I prefer to put in all lines `from . import ...`, but it's not so important.